### PR TITLE
Updated VehicleTypes csv and created new CSVs for VehicleMake, VehicleModel and VehicleBodyType

### PR DESCRIPTION
**Pull request by rawerner**
This PR contains data changes to the VehicleTypes CSV file as well as the creation of the VehicleMake, VehicleModel and VehicleBodyType csv files.

### CODE REVIEW

File: book-1-carnival-design/chapters/data/VehicleModel.csv [hunk 1]
```diff
@@ -0,0 +1,16 @@
+vehicle_make_id,name,,,,,,,,
```
Comment (stevebrownlee) [inline book-1-carnival-design/chapters/data/VehicleModel.csv:1]: This is weird. What's with all the extra commas?

Comment (rawerner) [reply book-1-carnival-design/chapters/data/VehicleModel.csv:1]: That is weird...I'll fix that.

File: book-1-carnival-design/chapters/data/VehicleModel.csv [hunk 2]
```diff
@@ -0,0 +1,16 @@
+vehicle_make_id,name
```
Comment (jisie) [inline book-1-carnival-design/chapters/data/VehicleModel.csv:1]: This needs to be `model_id`. I know this is minor but currently our FKs are called `make_id` and `model_id`. So I didn't know if we wanted to change the PK to reflect that? I don't think `vehicle` is necessary for both make and model.

Comment (rawerner) [reply book-1-carnival-design/chapters/data/VehicleModel.csv]: The only reason I added the 'vehicle' to the beginning of each table was that it groups related tables together when viewing the list of tables in pgAdmin. It's also good practice to be more explicit on what the data is. Anybody new to the database may ask which tables belong to vehicles and this makes it very clear. But I'm happy to change it back if you feel strongly.

Comment (rawerner) [reply book-1-carnival-design/chapters/data/VehicleModel.csv]: There is an error here that I'm fixing either way. It has vehicle_make_id in the Model file...eeeek. Sorry.

**Pull request merged**